.travis.yml

```diff
@@ -3,18 +3,7 @@ sudo: false
 bundler_args: ""
 cache: bundler
 rvm:
-  - 2.0.0
-  - 2.1.8
-  - 2.2.4
-  - 2.3.0
+  - 2.5.8
 gemfile:
-  - gemfiles/rails3.2.gemfile
-  - gemfiles/rails4.1.gemfile
   - gemfiles/rails4.2.gemfile
   - gemfiles/rails5.0.gemfile
```

Comment:
Should we include 5.1, 5.2 ?

Reply:
No, not yet. There was some hardcoded top level versions. Prolly nothing, but I'd rather address that out of band when we start on 5.1

Reply:
5.2 would be nice ... 5.0 can go away once we no longer use it :)